projects/go-dns/build.sh

```diff
@@ -21,6 +21,22 @@ function compile_fuzzer {
   function=$2
   fuzzer=$3
 
+  if [[ $SANITIZER = *coverage* ]]; then
```

Comment:
something like this needs to be done for each Go project, right?

Sadly, we decided not to put `compile_fuzzer` function anywhere globally and instead copy-pasted it into every Go project's build_file. Now is a moment to reconsider that decision

Reply:
Yes, this needs to be done for each project.

Do you want this generalization be part of this PR ? (or rather in another one)

Reply:
let's do it in another PR